apps/web/components/Organization/Wrapper/Themes/HveTheme/HveHeader.tsx

```diff
@@ -43,6 +46,14 @@ const HveHeader: React.FC<React.PropsWithChildren<HeaderProps>> = ({
 
   const screenWidth = getScreenWidthString(width)
 
+  const { activeLocale } = useI18n()
+  const logoAltText = n(
```

Comment:
Just in case there isn't a title for the organization perhaps it'd be better to fallback to an empty string instead of potentially concatenating undefined or '' with ' Frontpage'